server/constants/order_status.js

```diff
@@ -5,16 +5,20 @@
  * pending -> active (subscription)
  * pending -> active -> cancelled (subscription)
  * pending -> cancelled
- * pending -> rejected
  * pending -> expired
  */
 
 export default {
-  PENDING: 'PENDING',
-  PAID: 'PAID',
-  ACTIVE: 'ACTIVE',
-  CANCELLED: 'CANCELLED',
-  REJECTED: 'REJECTED',
-  ERROR: 'ERROR',
-  EXPIRED: 'EXPIRED',
+  NEW: 'NEW', // New default state since August 2020
+  REQUIRE_ACTION: 'REQUIRE_ACTION', // For Strong Customer Authentication ("3D Secure")
+  PAID: 'PAID', // For One Time Contributions
+  ERROR: 'ERROR', // For One Time and Recurring Contribution
+  // This is only for "Recurring Contributions"
+  ACTIVE: 'ACTIVE', // Active Recurring contribution with up to date payments
+  CANCELLED: 'CANCELLED', // When it's Cancelled by contributors or automatically after X failures
+  // This is only for "Manual" payments
+  PENDING: 'PENDING', // Initial state
```

Comment:
Not especially for this PR, but maybe we can take the opportunity to make it more explicit too with something like `WAITING_HOST_CONFIRMATION`. That might require more refactor tho.

Reply:
I don't want to touch too much the old "Transactions" tab.

Maybe we should do that when we work on the new "Transactions" tab in the Host Dashboard.